### ISSUE TITLE
About loss function

### ISSUE THREAD

**Issue by imsotable** [yshMars/DistilPose#1]
Hello, I have a question about the overall loss function，
![image](https://github.com/yshMars/DistilPose/assets/77181570/3bf865a3-1ede-434a-a3ee-77362679a49e)
What is the weight ratio of these loss functions?

**Comment by yshMars** [comment]
The hyper-parameters are set to α1 = α2 = 5e − 4, α3 = 1, α4 = 1e − 2.
We show the hyper-parameters in Section 4.1.2:
![image](https://github.com/yshMars/DistilPose/assets/37333697/cf228595-b79b-482e-8db7-ffc76bf15875)
And we also conduct an experiment to find out the best hyper-parameters:
![image](https://github.com/yshMars/DistilPose/assets/37333697/f63dd20c-8b38-4874-950a-609c2ee22470)
